templates/prescription/general_edit.html

```diff
@@ -277,23 +323,18 @@
     $('#hiddendiv').hide();
 }
 
+function iframetopardiv(string){
+    var name=string
+    document.getElementById('pharmacy').value=name;
+    $("#hiddendiv").html( "&nbsp;" );
+    $('#hiddendiv').hide();
+}
+
 function cancelParlookup () {
     $('#hiddendiv').hide();
     $("#hiddendiv").html( "&nbsp;" );
 }
 
-$().ready(function() {
-    $("#drug").autocomplete('library/ajax/prescription_drugname_lookup.php',
-                            {
-                            width: 200,
-                            scrollHeight: 100,
-                            selectFirst: true
-                            });
-    $("#drug").focus();
-    $("#prescribe").submit(function() { return CheckForErrors(this) });
-    $("#druglookup").click(function() { DoDrugLookup(this) });
```

Comment:
this does not break. Unfortunately, I built this so long ago that I have forgotten the why of much of this. I do know it works. I will test it today and report back.
I removed one of the auto-complete functions because the pharmacy stuff was moved out of this area in light of it could be assigned in the patient chart.  Was trying to do too much in one space. Tina suggested the move months ago. She was the cause of the confirmation page being built and the AJAX to send the code off to a processing server keeping the essential code hidden from prying eyes.